components/org.wso2.carbon.extension.identity.sso.jwt/src/main/java/org/wso2/carbon/identity/sso/jwt/factory/JWTInboundIdentityResponseFactory.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.wso2.carbon.identity.sso.jwt.factory;
+
+import org.apache.commons.lang.StringUtils;
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityResponse;
+import org.wso2.carbon.identity.application.authentication.framework.inbound.HttpIdentityResponseFactory;
+import org.wso2.carbon.identity.application.authentication.framework.inbound.IdentityResponse;
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+import org.wso2.carbon.identity.sso.jwt.message.JWTInboundResponse;
+import org.wso2.carbon.identity.sso.jwt.util.JWTInboundConstants;
+import org.wso2.carbon.identity.sso.jwt.util.JWTInboundUtil;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletResponse;
+
+/**
+ * This class represents a factory for JWT IdentityResponse instances which will result from the framework,
+ * after the authentication step.
+ *
+ * This class is extensible either by implementing the HttpIdentityResponseFactory or extending the
+ * JWTInboundIdentityResponseFactory. The customized class should be defined under
```

Comment:
As discussed during the review, let's remove this for the time being. The extension support implemented from OSGi service level

Reply:
fixed